network/discovery/dv5_filters.go

```diff
@@ -8,9 +8,9 @@ import (
 	"go.uber.org/zap"
 )
 
-// limitNodeFilter checks if limit exceeded
+// limitNodeFilter returns true if the limit is exceeded
 func (dvs *DiscV5Service) limitNodeFilter(node *enode.Node) bool {
-	return !dvs.conns.AtLimit(libp2pnetwork.DirOutbound)
+	return dvs.conns.AtLimit(libp2pnetwork.DirOutbound)
```

Comment:
I'm not sure about this change, I was under the impression filters that returns true means that peers is ok. And if we're at the limit shouldn't we filter out the peer? maybe I misunderstood filters?

Reply:
Oh okay, it made more sense to me for this function to return true if we were at the limit. But I'll revert it